packages/validator/src/util/params.ts

```diff
@@ -229,6 +229,7 @@ function getSpecCriticalParams(localConfig: ChainConfig): Record<keyof ConfigWit
     MAX_BLOB_COMMITMENTS_PER_BLOCK: denebForkRelevant,
     KZG_COMMITMENT_INCLUSION_PROOF_DEPTH: denebForkRelevant,
     MAX_BLOBS_PER_BLOCK: denebForkRelevant,
+    BLOB_SCHEDULE: denebForkRelevant,
```

Comment:
this will be problematic, if a beacon node does not yet have `BLOB_SCHEDULE` or didn't backport deneb/electra values then it won't be able to interop with lodestar vc which is really bad since Obol uses it as primary vc

I think we need to add this as `fuluForkRelevant` for now, we lose some checks on deneb/electra values but that should be fine, might make it stricter once we know majority/all bns implementing this, or earliest in the backport PR when we remove previous constants

Reply:
updated in https://github.com/ChainSafe/lodestar/pull/7729/commits/f3a03671b2dbd06afd3c4cd892b7a3a91e77ccbd, I don't see any other solution